website/sidebars.json

```diff
@@ -103,9 +103,14 @@
       }
     ],
     "Ethereum Compatibility": [
+      "evm/evm-network-status",
+      "evm/evm-simple",
+      "evm/near-pet-shop",
       "evm/near-web3-provider",
       "evm/truffle",
-      "evm/evm-local-setup"
+      "evm/evm-local-setup",
+      "evm/near-eth-rpc",
+      "evm/evm-testing"
```

Comment:
Is the double "evm" prefix on a lot of entries necessary? I get it that "evm-simple" is a project name, but can we change "evm/evm-" to "evm/" otherwise?

Reply:
Docusaurus is weird, and I wanted to make sure that those slugs aren't ever used again. I believe if we have `local-testing` somewhere else (even without the `evm/` part) it'll cause issues. It comes into play with the metadata at the top of the Markdown files like here:

```
---
id: evm-local-setup
…
```

Reply:
OK, too bad. Makes for strange URLs, which is a DevX problem: it's nice when URLs are so simple that one can just remember them directly. But that's a concern that's out of scope for this PR, certainly.